src/UpgradeAssistant.Mappings/UpgradeAssistant.Mappings.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <!-- Pack related properties -->
+  <PropertyGroup>
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.UpgradeAssistant.Mappings</PackageId>
+    <PackageVersion>1.0.0</PackageVersion>
```

Comment:
>PackageVersion [](http://example.com/codeflow?start=5&length=14)

is version always hardcoded to 1.0.0? it should have some auto increment from build system?

Reply:
I figured we'd bump it as we go. Is there a standard convention you suggest instead?